python/cuspatial/setup.py

```diff
@@ -39,6 +39,7 @@
         sources=cython_files,
         include_dirs=[
             "../../cpp/include/cuspatial",
+            "../../cpp/src",
```

Comment:
Why do you need to reach into the `src` here? The header you need should be under `include/cuspatial`, no?